integreat_cms/release_notes/current/unreleased/3166.yml

```diff
@@ -0,0 +1,2 @@
+en: Prevent the bad request error and show an warning when users try to upload more than 100 xliff files
+de: Verhindere den Fehler "Inkorrekte Anfrage" und zeige eine Warnung an, wenn Benutzer:innen versuchen, mehr als 100 xliff-Dateien hochzuladen
```

Comment:
I remember that we said we should use less technical terms and one given example was xliff. So maybe it's better to write "Übersetzungsdatei" instead of xliff-file, but it's not really relevant and I'm fine either way :)

Reply:
Applied :)